include/IECore/TransformationMatrix.h

```diff
@@ -87,6 +88,9 @@ class IECORE_EXPORT TransformationMatrix
 template<class T>
 std::ostream &operator << ( std::ostream &os, const TransformationMatrix<T> &x );
 
+template<class T>
+IECORE_API void murmurHashAppend( IECore::MurmurHash &h, const TransformationMatrix<T> &data );
```

Comment:
I think the `IECORE_API` here is the cause of this Windows build failure :

```
D:\a\cortex\cortex\cortex-pr1150-moreMurmurHash-2021_04_14_2143-3a1f45da-windows-python2\include\IECore\Spline.inl(376): error C2491: 'IECore::murmurHashAppend': definition of dllimport function not allowed
```

Where we're providing the function definition in a header (the `.inl` file here), I think we want to omit `IECORE_API` and replace it with `inline` instead.

Reply:
Fixed squashed into 467a02b